src/cli/helpers.py

```diff
@@ -217,15 +218,27 @@ def record_struct(name, search, search_type, record_links, save, category, subca
     language = "general"
     name = re.sub("[^A-Za-z0-9]+", "", name)
     name = name + "_file_check"
-    save_path = Path("assessments") / save / "evidence.json"
+    if environment == 'production':
+        save_path = Path("assessments/"+save+"/evidence.json")
+    else:
+        print('running as dev')
+        save_path = Path("assessments/"+save+"/evidence.json")
+    
     # save_path = "/rubric/" + language + "/" + name + ".json"
 
     # Load records
     try:
-        with open(save_path, "r") as records_file:
-            records = json.loads(records_file)
+        with open(save_path, "r")as records_file:
+            records_data = records_file.read()
+        records = json.loads(records_data)
```

Comment:
I believe `json.loads` is to load a json string, use `json.load` to load a json from a file.

Reply:
Was getting an issue with PATH being defined and being passed into load. It errors out. This works for now.